velox/functions/lib/string/StringImpl.h

```diff
@@ -427,6 +427,23 @@ FOLLY_ALWAYS_INLINE void toBase64Url(
   encoding::Base64::encodeUrl(input.data(), input.size(), output.data());
 }
 
+template <typename TOutString, typename TInString>
+FOLLY_ALWAYS_INLINE void fromBase64Url(
+    TOutString& output,
+    const TInString& input) {
+  try {
+    auto inputSize = input.size();
+    const bool hasPad =
+        (*(input.end()) == encoding::Base64::kBase64Pad) ? true : false;
+    output.resize(encoding::Base64::calculateDecodedSize(
+        input.data(), inputSize, hasPad));
```

Comment:
Nit : Maybe abstract a variable for input.data() also.

Reply:
done